keyboards/the_royal/liminal/config.h

```diff
@@ -0,0 +1,44 @@
+#pragma once
+
+#include "config_common.h"
+
+/* USB Device descriptor parameter */
+#define VENDOR_ID       0x1337
```

Comment:
This vender ID is owned already, please use a different VID. 

https://www.the-sz.com/products/usbid/index.php?v=0x1337&p=&n=